idx/memory/memory_test.go

```diff
@@ -96,6 +98,7 @@ func TestGetAddKey(t *testing.T) {
 }
 
 func TestFind(t *testing.T) {
+	log.NewLogger(0, "console", fmt.Sprintf(`{"level": %d, "formatting":false}`, 1))
```

Comment:
this is just left over from me debugging (it makes the log calls in memory.go work).   It should be removed.